src/autogluon/bench/runbenchmark.py

```diff
@@ -130,6 +126,34 @@ def run_benchmark(
     _dump_configs(benchmark_dir=benchmark.metrics_dir, configs=configs, file_name="configs.yaml")
 
     if configs.get("METRICS_BUCKET", None):
+        framework_name = module_kwargs["run_kwargs"]["framework"].split(":")[0]
+        upload_path = []
+        if module_name == "multimodal":
+            upload_path = [
+                configs.get("mode", None),
+                module_kwargs["run_kwargs"].get("dataset_name", None),
+                module_kwargs["run_kwargs"].get("constraint", None),
+                framework_name,
```

Comment:
We might want to add at least time_limit as well?

Reply:
`time_limit` is `constraint` here I believe
`constraint/time_limit` in AutoMM custom configs look like the one below:

```
g4_12x:
  INSTANCE: g4dn.12xlarge
  MAX_MACHINE_NUM: 200  
  BLOCK_DEVICE_VOLUME: 300  
  RESERVED_MEMORY_SIZE: 8000
  TIME_LIMIT: 3600
```

Reply:
You are right.